.env

```diff
@@ -4,6 +4,7 @@ AWS_SECRET_ACCESS_KEY=bar
 AWS_DEFAULT_REGION=eu-west-2
 BASM_API_BASE_URL=http://localhost:9999
 BASM_BUCKET_NAME=basm
+CRON_AUTOMATIC_LOCATION_MAPPING=-
```

Comment:
This is disabled by default in all envs as waiting on the new endpoint to become available on BaSM.